---
layout: page
title: Publications
permalink: /publications/
---
*lab members in bold*

(I am very bad at keeping this up to date. So you can always check [google scholar](https://scholar.google.com/citations?hl=en&user=gVwJZpAAAAAJ&view_op=list_works&sortby=pubdate))

## 2022

## 2021
- Guo, X., D. Fang, S. Kumar Sahu, S. Yang, X. Guang, R. Folk, **S. A. Smith**, A. S. Chanderbali, S. Chen, M. Liu, T. Yang, S. Zhang, X. Liu, X. Xu, P. S. Soltis, D. E. Soltis, and H. Liu. 2021. Chloranthus genome provides insights into the early diversification of angiosperms. **Nature Communications**. 12:1-14. [link](https://www.nature.com/articles/s41467-021-26922-4)

- Linan, A. G., J. A. Myers, C. E. Edwards, A. E. Zanne, **S. A. Smith**, G. Arellano, L. Cayola, W. Farfan-Ríos, A. F. Fuentes, K. García-Cabrera, S. González-Caro, M. I. Loza, M. J. Macía, Y. Malhi, B. Nieto-Ariza, N. Salinas, M. Silman, and J. S. Tello. 2021. The evolutionary assembly of forest communities along environmental gradients: recent diversification or sorting of pre-adapted clades? **New Phytologist**. [link](https://doi.org/10.1111/nph.17674)

- **Walker, J. F.**, **S. A. Smith**, **R. Hodel**, and E. Moyroud. 2021. Concordance-Based Approaches for the Inference of Relationships and Molecular Rates with Phylogenomic Data Sets. **Systematic Biology**. syab052. [link](https://doi.org/10.1093/sysbio/syab052)

- **Figueroa, H.**, **H. E. Marx**, M. B. de S. Cortez, C. J. Grady, N. J. Engle-Wrye, J. Beach, A. Stewart, R. A. Folk, D. E. Soltis, P. S. Soltis, and **S. A. Smith**. 2021. Contrasting patterns of phylogenetic diversity and alpine specialization across the alpine flora of the American mountain range system. **Alpine Botany**. [link](https://doi.org/10.1007/s00035-021-00261-y)

- Keppel, G., D. Craven, P. Weigelt, **S. A. Smith**, M. T. van der Sande, B. Sandel, S. C. Levin, H. Kreft, and T. M. Knight. 2021. Synthesizing tree biodiversity data to understand global patterns and processes of vegetation. **Journal of Vegetation Science**. 32:e13021.

- Cortez, M. B. De S. , R. A. Folk, C. J. Grady, J. P. Spoelhof, **S. A Smith**, D. E. Soltis, and P. S. Soltis. 2021. Is the age of plant communities predicted by the age, stability and soil composition of the underlying landscapes? An investigation of OCBILs. **Biological Journal of the Linnean Society**. 133:297-316.

- **Stull., G. W.**, X.-J. Qu, C. Parins-Fukuchi, Y.-Y. Yang, J.-B. Yang, Z.-Y. Yang, Y. Hu, H. Ma, P. S. Soltis, D. E. Soltis, D.-Z. Li, **S. A. Smith**, and T.-S. Yi. 2021. Gene duplications and genomic conflict underlie major pulses of phenotypic evolution in gymnosperms. **Nature Plants**. [link](https://www.biorxiv.org/content/10.1101/2021.03.13.435279v1.abstract)

- **Figueroa, H.** and **S. A. Smith**. 2021. A targeted phylogenetic approach helps explain New World functional diversity patterns of two eudicot lineages. **Journal of Biogeography**. 48: 202-215.

- Parins-Fukuchi, C., **G. W. Stull**, and **S. A. Smith**. 2021. Phylogenomic conflict coincides with rapid morphological innovation. **PNAS**. 118:e2023058118.

- Singhal, S., T. Colston, M. Grundler, **S. A. Smith**, G. C. Costa, G. Colli, C. Moritz, A. Pyron, and D. Rabosky. 2021. Congruence and conflict in the higher-level phylogenetics of squamate reptiles: an expanded phylogenomic perspective.**Systematic Biology**. 70:542-557 [link](https://doi.org/10.1093/sysbio/syaa054)

- Morales-Briones, D., G. Kadereit, D. T Tefarikis, M. J. Moore, **S. A. Smith**, S. F. Brockington, A. Timoneda, W. C. Yim, J. C Cushman, and Y. Yang. 2021. Disentangling Sources of Gene Tree Discordance in Phylogenomic Data Sets: Testing Ancient Hybridizations in Amaranthaceae s.l. **Systematic Biology**. 70:219-235.


## 2020
- **Smith, S. A., N. Walker-Hale**, and **J. F. Walker**. 2020. Intragenic Conflict in Phylogenomic Data Sets. **Molecular Biology and Evolution**. 37: 3380-3388.

- Mishler, B. D., R. Guralnick, P. S. Soltis, **S. A. Smith**, D. Soltis, N. Barve, J. M. Allen, and S. Laffan. 2020. Spatial phylogenetics of the North American flora. **Journal of Systematics and Evolution**. 58:393-405. 

- Lu, L., H. Hu, D. Peng, B. Liu, J. Ye, T. Yang, H. Li, M. Sun, **S. A. Smith**, P. S. Soltis, D. E. Soltis, Z. Chen. 2020. Noise does not equal bias in assessing the evolutionary history of the angiosperm flora of China: A response to Qian (2019). **Journal of Biogeography**. 47:2286-2291.

- Sandel, B., P. Weigelt, H. Kreft, G. Keppel, M. T van der Sande, S. Levin, **S. A. Smith**, D. Craven, and T. M. Knight. 2020. Current climate, isolation and history drive global patterns of tree phylogenetic endemism. **Global Ecology and Biogeography**. 29:4-15.

- **Stull, G. W.**, P. Soltis, D. Soltis, M. Gitzendanner, and **S. A. Smith**. 2020. Nuclear phylogenomic analyses of asterids conflict with plastome trees and support novel relationships among major lineages. **American Journal of Botany**. 107:790-805.

- **Smith, S. A.**, **N. Walker-Hale**, **J. F. Walker**, and **J. Brown**. 2020. Phylogenetic conflicts, combinability, and deep phylogenomics in plants. **Systematic Biology.** 69:579-592. [link](https://doi.org/10.1101/371930)

- **Larson, D. A., J. F. Walker, O. M. Vargas**, and **S. A. Smith**. 2020. A consensus phylogenomic approach highlights paleopolyploid and rapid radiation in the history of Ericales. **American Journal of Botany**. 107:773-789.

- Sheehan, H., T. Feng, **N. Walker‐Hale**, S. Lopez‐Nieves, B. Pucker, R. Guo, W. Yim, R. Badgami, A. Timoneda, **L. Zhao**, H. Tiley, D. Copetti, M. Sanderson, J. Cushman, M. Moore, **S. A. Smith**, and S. Brockington. 2020. Evolution of l‐DOPA 4, 5‐dioxygenase activity allows for recurrent specialisation to betalain pigmentation in Caryophyllales **New Phytologist**. 227:914-929.

## 2019
- Yao, G.,J. Jin, H. Li, J. Yang, V. Mandala, M. Croley, R. Mostow, N. Douglas, M. Chase, M. Christenhusz, D. Soltis, P. Soltis, **S. A. Smith**, S. Brockington, M. Moore, and T. Yi. 2019. Plastid phylogenomic insights into the evolution of Caryophyllales. **Molecular Phylogenetics and Evolution**. 134:74-86

- **Vargas, O. M**, M. Heuertz, **S. A. Smith**, and C. Dick. 2019. Target sequence capture in the Brazil nut family (Lecythidaceae): Marker selection and in silico capture from genome skimming data. ** Molecular Phylogenetics and Evolution**. 135:98-104

- **Chang, J.**, D. Rabosky, **S. A. Smith**, and M. Alfaro. 2019. An R package and online resource for macroevolutionary studies using the ray‐finned fish tree of life. **Methods in Ecology and Evolution**. 10:1118-1124.

- Allen, J., C. C Germain-Aubrey, N. Barve, K. M. Neubig, L. C. Majure, S. W. Laffan, B. D. Mishler, H. L. Owens, **S. A Smith**, W. M. Whitten, J. R. Abbott, D. E. Soltis, R. Guralnick, and P. S. Soltis. 2019. Spatial Phylogenetics of Florida Vascular Plants: The Effects of Calibration and Uncertainty on Diversity Estimates. **iScience**. 11:57-70.

- **Smith, S. A.** and **J. F. Walker**. 2019. PyPHLAWD: A python tool for phylogenetic dataset construction. **Methods in Ecology and Evolution**. 10:104-108. [link](https://doi.org/10.1111/2041-210X.13096)

- **Wang, N.**, **Y. Yang**, M. J. Moore, S. F. Brockington, **J. F. Walker**, **J. W. Brown**, B. Liang, T. Feng, C. Edwards, J. Mikenas, J. Olivieri, V. Hutchinson, A. Timoneda, T. Stoughton, R. Puente, L. C. Majure, U. Eggli, **S. A. Smith**. 2018. Evolution of Portulacineae marked by gene tree conflict and gene family expansion associated with adaptation to harsh environments. Molecular Biology and Evolution

- Allen, J., C. C Germain-Aubrey, N. Barve, K. M. Neubig, L. C. Majure, S. W. Laffan, B. D. Mishler, H. L. Owens, **S. A Smith**, W. M. Whitten, J. R. Abbott, D. E. Soltis, R. Guralnick, P. S. Soltis. 2019. Spatial Phylogenetics of Florida Vascular Plants: The Effects of Calibration and Uncertainty on Diversity Estimates. iScience. 11:57-70.

## 2018
- **Walker, J.**, **J. W. Brown**, and **S. A. Smith**. 2018. Analyzing contentious relationships and outlier genes in phylogenomics. Systematic Biology. 67:916–924. [link](https://doi.org/10.1093/sysbio/syy043)
- **Smith, S. A.** and **J. F. Walker**. 2018. PyPHLAWD: A python tool for phylogenetic dataset construction. Methods in Ecology and Evolution [link](https://doi.org/10.1111/2041-210X.13096)
- Li-Min, L., L. Mao, T. Yang, J. Ye, B. Liu, H. Li, M. Sun, J. T. Miller, S. Mathews, H. Hu, Y. Niu, D. Peng, Y. Chen, **S. A. Smith**, M. Chen, K. Xiang, C. Le, V. Dang, A. Lu, P. S. Soltis, D. E. Soltis, J. Li and Z. Chen. 2018. Evolutionary history of the angiosperm flora of China-cradle vs. museum. Nature. 554:234-238. doi:10.1038/nature25485
- **Smith, S. A.**, **J. F. Walker**, and **J. Brown**. 2018. So many genes, so little time: rate variation in phylogenomic datasets. PLoS One [link](https://doi.org/10.1101/114975)
- **Smith, S. A.** and **J. W. Brown**. 2018. Constructing a comprehensive seed plant phylogeny. American Journal of Botany [link](http://onlinelibrary.wiley.com/doi/10.1002/ajb2.1019/full)
- **Pease J.B.**, **J. W. Brown**, **J. F. Walker**, **C. E. Hinchliff**, **S. A. Smith**. 2018. Quartet Sampling distinguishes lack of support from conflicting support in the plant tree of life. American Journal of Botany [link](onlinelibrary.wiley.com/doi/10.1002/ajb2.1016/abstract)
- Cheng, S., M. Melkonian, **S. A. Smith**, S. Brockington, J. M. Archibald, P. Delaux, F. Li, B. Melkonian, E. V. Mavrodiev, W. Sun, Y. Fu, H. Yang, D. E. Soltis, S. W. Graham, P. S. Soltis, X. Liu, X. Xu, G. K. Wong. 2018. 10KP: A phylodiverse genome sequencing plan. GigaScience
- Eiserhardt, W. L., A. Antonelli, D. J. Bennett, L. R. Botigué, J. G. Burleigh, S. Dodsworth, B. J. Enquist, F. Forest, J. T. Kim, A. M. Kozlov, I. J. Leitch, B. S. Maitner, S. Mirarab, W. H. Piel, O. A. Pérez‐Escobar, L. Pokorny, C. Rahbek, B. Sandel, **S. A Smith**, A. Stamatakis, R. A. Vos, T. Warnow, W. J. Baker. 2018. A roadmap for global synthesis of the plant tree of life. American Journal of Botany


## 2017
- Fleischmann, A., J. Schlauer, T. Givnish, and S. A. Smith. 2017. Evolution of carnivory in angiosperms. Chapter 3. in forthcoming book on Carnivory in angiosperms.
- Lopez-Nieves, S., Y. Yang, T. Feng, S. A. Smith, S. F. Brockington, and H. A. Maeda. 2017. Relaxation of Tyrosine Pathway Regulation Underlies the Evolution of Betalain Pigmentation in Caryophyllales. New Phytologist
- Brown, J. W., N. Wang, and S. A. Smith. 2017. The development of scientific consensus: analyzing conflict and concordance among Avian phylogenies. Molecular Phylogenetics and Evolution [link](https://doi.org/10.1101/123034)
- Brown, J. W. and S. A. Smith. 2017. The Past Sure Is Tense: On Interpreting Phylogenetic Divergence Time Estimates. Systematic Biology [link](https://doi.org/10.1101/113720)
- Walker, J., Y. Yang, M. Moore, S. F. Brockington, and S. A. Smith. 2017. Conflict among carnivores: whole transcriptomes unable to resolve the carnivorous clade of Caryophyllales. American Journal of Botany. [link](https://doi.org/10.1101/115741)
- Smith, S. A., J. W. Brown, Y. Yang, R. Bruenn, C. P. Drummond, S. F. Brockington, J. F. Walker, N. Last, N. A. Douglas, M. J. Moore. 2017. Disparity, diversity, and duplications in the Caryophyllales. New Phytologist. link
- Yang, Y., M. J. Moore, S. F. Brockington, J. Mikenas, and S. A. Smith. 2017. Improved transcriptome sampling pinpoints widespread paleopolyploidy events in Caryophyllales New Phytologist. [link](https://doi.org/10.1101/143529)
- Brown, J., J. Walker, and S. A. Smith. 2017. phyx: Phylogenetic tools for Unix. Bioinformatics. [link](https://doi.org/10.1093/bioinformatics/btx063)
- Yang, Y., M. J. Moore, S. F. Brockington, A. Timoneda-Monfort, T. Feng, H. E. Marx, Joseph F. Walker, and S. A. Smith. 2017. An efficient field and laboratory workflow for plant phylotranscriptomic projects. Applications in Plant Sciences. [link](https://doi.org/10.1101/ 079582)

## 2016

- Smith, S.A.* and J. Pease. 2016. Heterogeneous molecular processes among the causes of how sequence similarity scores can fail to recapitulate phylogeny Briefings in Bioinformatics [link](http://bib.oxfordjournals.org/content/early/2016/04/20/bib.bbw034.full)
- O'Meara, B. C., S. D. Smith, W. S. Armbruster, L. D. Harder, C. R. Hardy, L. C. Hileman, Larry Hufford, Amy Litt, Susana Magallón, S. A. Smith, P. F. Stevens, C. B. Fenster, P. K. Diggle. Non-equilibrium dynamics and floral trait interactions shape extant angiosperm diversity.Proc Roy Soc B [link](http://rspb.royalsocietypublishing.org/content/283/1830/20152304)

## 2015

- C. E. Hinchliff*, S. A. Smith*+, J. F. Allman, J. G. Burleigh, R. Chaudhary, L. M. Coghill, K. A. Crandall, J. Deng, B. T. Drew, R. Gazis, K. Gude, D. S. Hibbett, L. A. Katz, H. D. Laughinghouse IV, E. J. McTavish, P. E. Midford, C. L. Owen, R. H. Ree, J. A. Rees, D. E. Soltis, T. Williams, and K. A. Cranston+. Synthesis of phylogeny and taxonomy into a comprehensive tree of life. * Co-first authors, +corresponding authors. PNAS link
- Smith, S. A., M. Moore, J. W. Brown, and Y. Yang (2015). Analysis of phylogenomic datasets reveals conflict, concordance, and gene duplications with examples from animals and plants. BMC Evolutionary Biology link
- Brockington S.F.\*, Y. Yang\*, F. Gandia-Herrero, S. Covshoff, J.M. Hibberd, R.F. Sage, G.K.-S. Wong, M.J. Moore and S.A. Smith. Lineage specific gene radiations underly the evolution of novel betalain pigmentation in Caryophyllales. \*Co-first author. New Phytologist. link.
- Yang, Y. M.J. Moore, S.F. Brockington, D.E. Soltis, G.K.-S. Wong, E.J. Carpenter, Y. Zhang, L. Chen, Z.-X. Yan, Y.-L. Xie, R.F. Sage, S. Covshoff, J.M. Hibberd, M.N. Nelson, and S.A. Smith. Dissecting molecular evolution in the highly diverse plant clade Caryophyllales using transcriptome sequencing. Molecular Biology and Evolution. link.
- McTavish, E. J., C. E. Hinchliff, J. F. Allman, J. W. Brown, K. A. Cranston, M. T. Holder, J. A. Rees, and S. A. Smith. Phylesystem: a git-based data store for community curated phylogenetic estimates. Bioinformatics Accepted

## 2014

- Smith, S. A. and A. Stamatakis. 2014. Inferring and Postprocessing Huge Phylogenies. in Biological Knowledge Discovery Handbook: Preprocessing, Mining and Post-processing of Biological Data. Wiley, NY.
- Yang, Y. and S. A. Smith. 2014. Orthology Inference in Nonmodel Organisms Using Transcriptomes and Low-Coverage Genomes: Improving Accuracy and Matrix Occupancy for Phylogenomics MBE. link
- Hinchliff, C. and S. A. Smith. 2014. Some Limitations of Public Sequence Data for Phylogenetic Inference (in Plants) Plos ONE. link
- Izquierdo-Carrasco, F., J. Cazes, S. A. Smith, and A. Stamatakis. 2014. PUmPER: phylogenies updated perpetually. Bioinformatics. doi: 10.1093/bioinformatics/btu053 link
- Cornwell, W. K., M. Westoby, D. S. Falster, R. G. FitzJohn, B. C. O’Meara, M. W. Pennell, D. J. McGlinn, J. M. Eastman, A. T. Moles, P. B. Reich, D. C. Tank, I. J. Wright, Lonnie Aarssen, J. M. Beaulieu, R. M. Kooyman, M. R. Leishman, E. T. Miller, U. Niinemets, J. Oleksyn, A. Ordonez, D. L. Royer, S. A. Smith, P. F. Stevens, L. Warman, P. Wilf, and A. E. Zanne. 2014. Functional distinctiveness of major plant lineages. Journal of Ecology. 102: 345-356 link
- Drew, B. T., B. R. Ruhfel, S. A. Smith, M. J. Moore, B. G. Briggs, M. A. Gitzendanner, P. S. Soltis, and D. E. Soltis. 2014. Another look at the root of angiosperms reveals a familiar tale. Systematic Biology. doi: 10.1093/sysbio/syt108 link

## 2013

- S. A. Smith, J. Brown, and C. Hinchliff. 2013. Analyzing and Synthesizing Phylogenies Using Tree Alignment Graphs. PLoS Computational Biology. link
- Yang, Y. and S. A. Smith. 2013. Optimizing de novo assembly of short-read RNA-seq data for phylogenomics. BMC Genomics. link
- Rabosky, D. L., F. Santini, J. Eastman, S. A. Smith, B. Sidlauskas, J. Chang, and M. E. Alfaro. 2013 Rates of speciation and morphological evolution are correlated across the largest vertebrate radiation. Nat. Commun. 4:1958, doi: 10.1038/ncomms2958. link
- Ryan, J., P. Pang, C. Schitzler, A. Nguyen, R. Moreland, D. Simmons, B. Koch, W. Francis, P. Havlak, NISC Comparative Sequence Program, S. A. Smith , N. Putnam, S. Haddock, C. Dunn, T. Wolfsberg, J. Mullikin, M. Martindale, A. Baxevannis. 2013. The genome of the Ctenophore Mnemiopsis leidyi and its implications for cell type evolution. Science 342 link
- Zanne, A., D. C. Tank, W. K. Cornwell, J. M. Eastman, S. A. Smith, R. G. FitzJohn, D. J. McGlinn, B. C. OMeara, A. T. Moles, P. B. Reich, D. L. Royer, D. E. Soltis, P. F. Stevens, Mark Westoby, I. J. Wright, Lonnie Aarssen, R. I. Bertin, Andre Calaminus, R. Govaerts, F. Hemmings, M.
- R. Leishman, Jacek Oleksyn, P. S. Soltis, N. G. Swenson, L. Warman, J. M. Beaulieu. 2013. Three keys to the radiation of angiosperm into freezing environments. Nature. doi:10.1038/nature12872 link

## 2012

- Smith, S. A. and B. O'Meara. 2012. treePL: Divergence time estimation using penalized likelihood for large phylogenies. Bioinformatics. doi: 10.1093/bioinformatics/bts492. link
- Stamatakis, A., A. Aberer, C. Goll, S. A. Smith, S. Berger, and F. Izquierdo-Carrasco. 2012. Raxml-light: a tool for computing terabyte phylogenies. Bioinformatics, 28(15):2064–2066. doi: 10.1093/bioinformatics/bts309. link
- Grass Phylogeny Working Group II. 2012. New grass phylogeny resolves deep evolutionary relationships and discovers C4 origins. New Phytologist. 193: 304-312.

## 2011

- Izquierdo-Carrasco F, S. A. Smith, A. Stamatakis. 2011. Algorithms, Data Structures, and Numerics for Likelihood-based Phylogenetic Inference of Huge Trees. BMC Bioinformatics. 12(1):470. link
- Smith, S. A., N. G. Wilson, F. E. Goetz, C. Feehery, S. C. S. Andrade, G. W. Rouse, G. Giribet, C. W. Dunn. 2011. Resolving the evolutionary relationships of molluscs with phylogenomic tools. Nature pdf
- Soltis, D. E., S. A. Smith, N. Cellinese, et al. 2011. Angiosperm phylogeny: 17 genes, 640 taxa . American J. of Botany.98:704.
- Smith, S. A., J. Beaulieu, A. Stamatakis, and M. J. Donoghue. 2011. Understanding angiosperm diversification using large and small phylogenies. American J. of Botany. 98: 404-414. pdf
- S. Siebert, M. Robinson, S. Tintori, F. Goetz, R. Helm, S. A. Smith, N. Shaner, S. Haddock, C. Dunn. 2011. Differential Gene Expression in the Siphonophore Nanomia bijuga (Cnidaria) Assessed with Multiple Next-Generation Sequencing Workflows. PLoS ONE link

## 2010

- Smith, S. A. and M. J. Donoghue. 2010. Combining Historical Biogeography with Niche Modeling in the Caprifolium Clade of Lonicera (Caprifoliaceae, Dipsacales). Syst. Biol. 590: 322-341. pdf
- E. E. Goldberg,1 J. R. Kohn, R. Lande, K. A. Robertson, S. A. Smith, and B. Igic. 2010. Species Selection Maintains Self-Incompatibility. Science. 328: 587-591. pdf
- Edwards, E.J., C. P. Osborne, C. A. E. Strömberg, S. A. Smith, and C4 Grasses Consortium. 2010. The origins of C4 grasslands: integrating evolutionary and ecosystem science. Science. 328: 587-591. pdf
- Smith, S. A, J. M. Beaulieu and M. J. Donoghue. 2010. An uncorrelated relaxed-clock analysis suggests an earlier origin for flowering plants. PNAS. 107: 5897-5902. pdf (F1000)
- E. Edwards and S. A. Smith. 2010. Phylogenetic analyses reveal the shady history of C4 grasses. PNAS. 107: 2532-2538. pdf

## 2009

- Smith, S. A. and J. Beaulieu 2009. Life history influences rates of climatic niche evolution in flowering plants. Proc Roy Soc B. 276: 4345-4352. pdf
- Smith, S. A. and B. C. O'Meara 2009. Morphogenera, monophyly, and macroevolution. PNAS. 106: E97-E98. pdf
- Smith, S. A., J. Beaulieu, and M. J. Donoghue. 2009.  Mega-phylogeny approach for comparative biology: an alternative to supertree and supermatrix approaches. BMC Evol Biol. . 9: 37.link (listed as "Highly accessed") (F1000)
- Smith, S. A. 2009. Taking into account phylogenetic and divergence-time uncertainty in a parametric biogeographic analysis of the Northern Hemisphere plant clade Caprifolieae. Journal of Biogeography. pdf
- Cellinese, N., S. A. Smith, E. J. Edwards, S. T. Kim, R. C. Haberle, M. Avramakis and M. J. Donoghue. 2009. Historical biogeography of the endemic Campanulaceae of Crete. Journal of Biogeography. 36: 1253-1269. link
- Evans, M. E., S. A. Smith, R. E. Flynn and M. J. Donoghue. 2009. "Climate, niche evolution, and diversification of the "bird-cage" evening primroses (Oenothera, sections Anogra and Kleinia). American Naturalist. 173: 225-240. pdf

## 2008

- Smith, S. A. and M. J. Donoghue. 2008. Rates of Molecular Evolution are Linked to Life History in Flowering Plants. Science. 322: 86-89.pdf (F1000)
- Smith, S. A. and C. Dunn. 2008. Phyutility: a phyloinformatics utility for trees, alignments, and molecular data. Bioinformatics. 24: 715-716. pdf
- Ree, R. H. and S. A. Smith. 2008. Maximum-likelihood Inference of Geographic Range Evolution by Dispersal, Local Extinction, and Cladogenesis. Systematic Biology. 57: 4-14. pdf
- Smith, S. A. et al. 2008. Bioactive endophytes warrant intensified exploration and conservation. PLoS ONE. 3: e3052. pdf
- Dunn, C. W., A. Hejnol, D. Q. Matus, K. Pang, W. E. Browne, S. A. Smith, et al. 2008. Broad taxon sampling improves resolution of the Animal Tree of Life in phylogenomic analyses. Nature. 452: 745-749. pdf

## before 2008

- Roulston, T. H., S. A. Smith and A. L. Brewster. 2007. Comparison of Pan Trap and Intensive Net Sampling Techniques for Documenting a Bee (Hymenoptera: Apiformes) Fauna. Journal Kansas Entomological Society.
- Moore, B. R., S. A. Smith, M. J. Donoghue. 2006. Increasing Data Transparency and Estimating Phylogenetic Uncertainty in Supertrees: Approaches Using Nonparametric Bootstrapping. Systematic Biology 55:4. 662 - 676. pdf
- Donoghue, M.J. and S. A. Smith. 2004. Patterns in the Assembly of Termperate Forests Around the Northern Hemisphere. Philosophical Transactions of the Royal Society of London B. 359(1450):1633-44 pdf

